examples/multi-engine-setup/js/face-attachment-point-example.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * FaceAttachmentPointExample
+ * A basic example demonstrating how to attach object to a face feature
+ */
+import {Component, Object as WLEObject} from '@wonderlandengine/api';
+import {property} from '@wonderlandengine/api/decorators.js';
+
+import {ARFaceTrackingCamera} from '@wonderlandengine/8thwall-tracking';
+
+const ATTACHMENT_POINTS = [
+    'forehead',
+    'rightEyebrowInner',
+    'rightEyebrowMiddle',
+    'rightEyebrowOuter',
+    'leftEyebrowInner',
+    'leftEyebrowMiddle',
+    'leftEyebrowOuter',
+    'leftEar',
+    'rightEar',
+    'leftCheek',
+    'rightCheek',
+    'noseBridge',
+    'noseTip',
+    'leftEye',
+    'rightEye',
+    'leftEyeOuterCorner',
+    'rightEyeOuterCorner',
+    'upperLip',
+    'lowerLip',
+    'mouth',
+    'mouthRightCorner',
+    'mouthLeftCorner',
+    'chin',
+];
+
+export class FaceAttachmentPointExample extends Component {
+    public static TypeName = 'face-attachment-point-example';
+
+    /**
+     * The ARFaceTrackingCamera somewhere in the scene
+     */
+    @property.object()
+    ARFaceTrackingCamera!: WLEObject;
+
+    /**
+     * To which feature of the face should the object be attached
+     */
+    @property.enum(ATTACHMENT_POINTS)
+    attachmentPoint: number = 0;
+
+    /**
+     * Object which should be attached to the face feature
+     */
+    @property.object()
+    attachedObject!: WLEObject;
+
+    start() {
+        if (!this.ARFaceTrackingCamera) {
+            console.warn(
+                `${this.object.name}/${this.type} requires a ${ARFaceTrackingCamera.TypeName}`
+            );
+            return;
+        }
+        const camera = this.ARFaceTrackingCamera.getComponent(ARFaceTrackingCamera);
+        if (!camera) {
+            throw new Error(
+                `${ARFaceTrackingCamera.TypeName} was not found on ARFaceTrackingCamera`
+            );
+        }
+
+        // allocate some arrays
+        const cachedPosition = new Array<number>(3);
+        const cachedRotation = new Array<number>(4);
+        const cachedScale = [0, 0, 0];
```

Comment:
Prefer `Float32Array`, slightly less memory use (32-bit instead of 64-bit floats) and supposedly slightly faster to access (but minimally slower to constructor)